packages/blade/tokens/global/spacing.ts

```diff
@@ -0,0 +1,29 @@
+type spacing = Readonly<{
```

Comment:
Can we use `spacer` instead of `spacing` here?

Reply:
Umm it might become misleading since it's a scale. The actual component can be named as `spacer` since it seems like an acting verb.